charger/proxy.go

```diff
@@ -0,0 +1,40 @@
+package charger
+
+import (
+	"fmt"
+
+	"github.com/evcc-io/evcc/api"
+	"github.com/evcc-io/evcc/templates"
+	"gopkg.in/yaml.v3"
+)
+
+func init() {
+	registry.Add("template", NewChargerFromTemplateConfig)
+}
+
+func NewChargerFromTemplateConfig(other map[string]interface{}) (api.Charger, error) {
+	name := other["template"].(string)
```

Comment:
ich würde wieder mapstructure nehmen- immer gleiche Lösungen für gleiches Problem!

Reply:
Hast einen Tip wie die anderen Parameter außer `template` beim dekodieren ignoriert werden können? Hier ist nur `template` relevant, der Rest ist ja bei jedem Template anders

Reply:
Ich bräuchte wohl ein `util.DecodeOther` aber mit `ErrorUnused: false`

Reply:
Genau- den Rest in einen Sammler schieben:

```
type qualifiedConfig struct {
	Name, Type string
	Other      map[string]interface{} `mapstructure:",remain"`
}
```

Der Trick ist das `mapstructure:",remain"`